Automattic/jetpack
# Admin menu: add link to Customizer in sites with block-based theme an…

Fixes https://github.com/Automattic/vulcan/issues/355

## Proposed changes:

Until last year, Jetpack provided a way for sites with a block-based theme to include Custom CSS (defined in the Customizer). This feature [was removed from the UI](https://github.com/Automattic/jetpack/pull/31413) but still existed for backward compatibility. As a consequence, site owners of these sites had no visible way of editing existing Custom CSS.

This PR fixes it by ensuring that sites with a block-based theme and custom CSS have a link to the Customizer in the admin menu.
<img width="325" alt="Screenshot 2024-06-06 at 4 18 21 PM" src="https://github.com/Automattic/jetpack/assets/1620183/5f8cf95e-4944-4fa5-a5d1-c2742c12bc04">

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a Jetpack test site with this branch
- Ensure the site has a block-based theme (e.g. Twenty Twenty-Four)
- Activate the Custom CSS feature from `/wp-admin/admin.php?page=jetpack_modules`
- In the Customizer (`/wp-admin/customize.php`), add some custom CSS (_Additional CSS_ section) and publish it. Leave this tab open.
- From a new tab, deactivate the Custom CSS feature
- Verify that in the admin menu the _Appearance_ item contains an _Additional CSS_ subitem that links to the appropriate section in the Customizer
- Back in the Customizer tab, temporarily remove the custom CSS and publish the changes
- Verify that the _Additional CSS_ item is no longer present under _Appearance_  in the admin menu
